src/SearchBar.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
<SearchBar /> should also be added to the App.tsx file. And on click to search button should disappear and show TrainConnectionCard instead